pkg/proxy/endpoint_test.go

```diff
@@ -35,46 +35,51 @@ func TestClientHandler(t *testing.T) {
 	}))
 	defer ts.Close()
 
+	rcvCtx := eventContext{}
 	rcvBody := ""
 	rcvForwardURL := ""
-	rcvWebhookID := ""
-	rcvWebhookConversationID := ""
 	client := NewEndpointClient(
 		ts.URL,
 		[]string{" Host:       hostname", "customHeader:customHeaderValue", "customHeader2:       customHeaderValue 2",
 			"emptyHeader:", ":", "::", "removeControlCharacters:	tab"}, // custom headers
 		false,
 		[]string{"*"},
 		&EndpointConfig{
-			ResponseHandler: EndpointResponseHandlerFunc(func(webhookID, webhookConversationID, forwardURL string, resp *http.Response) {
+			ResponseHandler: EndpointResponseHandlerFunc(func(evtCtx eventContext, forwardURL string, resp *http.Response) {
 				buf, err := ioutil.ReadAll(resp.Body)
 				require.Nil(t, err)
 
+				rcvCtx = evtCtx
```

Comment:
nit: is there a need to rename this variable? (I don't have strong feelings here so feel free to drop).

Reply:
Yes, it's how we verify that the context was correctly passed to the handler function.